p/scripts/main.js

```diff
@@ -973,7 +973,8 @@ function init_shortcuts() {
 	});
 
 	document.addEventListener('keydown', ev => {
-		if (ev.target.closest('input, textarea') ||
+		if (ev.target.closest('input, select, textarea') ||
+				(document.activeElement && document.activeElement.isContentEditable) ||
```

Comment:
We might not need that line as I do not think we have anything editable

Reply:
What about search input? Will the redirect to the shortcuts page not be performed if the user types the '?' character?